build/squire-raw.js

```diff
@@ -1626,16 +1629,12 @@ var firstOrLastLine = function(self){
     var rect = range.getBoundingClientRect()
     var nodeOffset = rect.top
     var parentOffset = parentBlock.getBoundingClientRect().top
-    if(nodeOffset === parentOffset){
-      return {firstLine: true, lastLine: true}
-    }
     nodeOffset = nodeOffset - parentOffset
```

Comment:
This condition doesn't make sense to me - the offset of a node relative to it's parent doesn't, to my eyes, tell us anything about whether we're on the first or last line. It's leading to bugs, and, in testing it out, I can't find any cases in which leaving it out has any negative consequences. Am I missing some special case here?